src/oiiotool/oiiotool.cpp

```diff
@@ -3420,6 +3420,17 @@ OIIOTOOL_OP(warp, 1, [&](OiiotoolOp& op, span<ImageBuf*> img) {
     return ok;
 });
 
+// --demosaic
+OIIOTOOL_OP(demosaic, 1, [&](OiiotoolOp& op, span<ImageBuf*> img) {
+    std::string pattern   = op.options().get_string("pattern");
+    std::string algorithm = op.options().get_string("algorithm");
+    std::string layout    = op.options().get_string("layout");
+
+    return ImageBufAlgo::demosaic(*img[0], *img[1],
+                                  { { "pattern", pattern },
+                                    { "algorithm", algorithm },
+                                    { "layout", layout } });
+});
```

Comment:
Nice! I'm very pleased by how compact it was to turn it directly into an exposed oiiotool action.